src/openforms/registrations/tests/test_registration_hook.py

```diff
@@ -34,6 +35,7 @@ class ResultSerializer(serializers.Serializer):
     external_id = serializers.UUIDField()
 
 
+@override_settings(CELERY_TASK_ALWAYS_EAGER=True)
```

Comment:
I went for this because of the "retry flag" check. Since this happens at the end of the chain, if I only call the `register_submission` task then it doesn't get set :thinking: but I am a bit unhappy with how the flag is being set now, so maybe I need to rethink that :S